libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -35,6 +38,14 @@ export class PayPalManager {
     };
   }
 
+  async getCustomerPayPalSubscriptions(customer: Stripe.Customer) {
+    return customer.subscriptions?.data.filter(
```

Comment:
I left a similar comment on the other PR, but I just remembered that in the ADR 0040-stripe-typings, the team decided to never use expanded properties. I believe that applies to `customer.subscriptions` as well.